core/jreleaser-model/src/main/java/org/jreleaser/model/Changelog.java

```diff
@@ -56,6 +56,7 @@ public class Changelog extends AbstractModelObject<Changelog> implements Domain,
 
     private Boolean enabled;
     private Boolean links;
+    private Boolean skipMergeCommits;
```

Comment:
Awesome, but this should be set to `true` by default, since including merge commits should be the exceptional case when generating changelogs

Reply:
If it were to be set to `true` by default then it would break compatibility on the 1.x series, prompting for an immediate 2.0 release.

Reply:
Maybe something to consider for the 2.x release 😉

Reply:
@aalmiray when is the next release coming? Can't wait to use this feature ASAP :)